cli/ttlog/logger.go

```diff
@@ -102,12 +111,22 @@ func (logger *fileLogger) GetOpts() LoggerOpts {
 	return logger.opts
 }
 
+// Write implements io.Writer interface
+func (logger *fileLogger) Write(p []byte) (int, error) {
+	logger.mu.Lock()
+	defer logger.mu.Unlock()
+	return logger.Logger.Writer().Write(p)
```

Comment:
It is more canonical formatting.
```suggestion
	defer logger.mu.Unlock()

	return logger.Logger.Writer().Write(p)
```

Reply:
Corrected.